Project.toml

```diff
@@ -4,18 +4,15 @@ repo = "https://github.com/JuliaPolyhedra/LRSLib.jl.git"
 version = "0.5.0"
 
 [deps]
-BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
-Homebrew = "d9be37ee-ecc9-5288-90f1-b9ca67657a75"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Polyhedra = "67491407-f73d-577b-9b50-8179a7c68029"
+lrslib_jll = "3873f7d0-7b7c-52c3-bdf4-8ab39b8f337a"
 
 [compat]
-BinDeps = "0.7, 0.8, 1"
-Homebrew = "0.7"
 Polyhedra = "0.6.8"
-julia = "1"
+julia = "1.3"
```

Comment:
Can you add a Compat to fix the `jll` to its current version ? If a new version is released then we don't want it to automatically update for users as we may need to make change to LRSLib.jl

Reply:
Yep, but this needs to wait until https://github.com/JuliaPackaging/Yggdrasil/pull/2875 to be done correctly

Reply:
```suggestion
julia = "1.3"
lrslib_jll = "0.2.0"
```

Reply:
Done